src/Space.cpp

```diff
@@ -943,9 +943,7 @@ static void hitCallback(CollisionContact *c)
 		*/
 		const float threshold = 0.005;
 
-		const Aabb &aabb = mover->GetAabb();
-
-		vector3d correction = std::min(std::max(c->depth - threshold, 0.0) * c->timestep, c->depth + aabb.min.y + threshold) * c->normal;
+		vector3d correction = std::min(std::max(c->depth - threshold, 0.0) * c->timestep, c->depth + threshold) * c->normal;
```

Comment:
I need to check but won't this mean that other ships now either don't sit correctly on the pad, or won't look correct landing?

This might be the ship that needs fixing, not the code.

Reply:
![Screenshot from 2019-06-01 13-26-33](https://user-images.githubusercontent.com/13293916/58747871-5ee8d980-8471-11e9-8f25-f2b10b04e9f9.png)

IIRC there were some issue due to shields meshes but when you are landed  then collision test is disabled.

Anyway I just landed at 1x, 10x and 100x : it seems no issue here...

Reply:
PS: `AAbb.min.y` is negative, thus breaking things because `std::max` returned a negative value.